### ISSUE TITLE
Update dependent AWS libraries to support AWSSDK.Core >= 3.5.0

### ISSUE THREAD

**Issue by thesuavehog** [aws/aws-dotnet-extensions-configuration#74]
AWSSDK.Core is updated to v3.5.1, AWSSDK.SimpleSystemsManagement & AWSSDK.Extensions.NETCore.Setup library dependencies in this library are ~3.3.100.8 & ~3.3.100.1 (respectively) but those versions depend on AWSSDK.Core < 3.4.0 which means when you upgrade OTHER AWS libraries to use v3.5.1 of the AWSSDK.Core you get a warning in Visual Studio that you've got libraries depending on different versions of AWSSDK.Core.

## Describe the Feature
Update dependencies to new(est/er) versions which link to AWSSDK.Core v3.5+

### Is your Feature Request related to a problem?
Visual Studio warns:

```
Detected package version outside of dependency constraint: AWSSDK.SimpleSystemsManagement 3.3.100.8 requires AWSSDK.Core (>= 3.3.100.7 && < 3.4.0) but version AWSSDK.Core 3.5.1 was resolved.
Detected package version outside of dependency constraint: AWSSDK.Extensions.NETCore.Setup 3.3.100.1 requires AWSSDK.Core (>= 3.3.100 && < 3.4.0) but version AWSSDK.Core 3.5.1 was resolved.
```

## Proposed Solution
Update library dependency versions.

### Environment
I'm seeing this on Windows in Visual Studio but I presume it's not specific to my environment/platform.

* [ ] :wave: I may be able to implement this feature request
* [X] :warning: This feature might incur a breaking change  

---

This is a :rocket: Feature Request

**Comment by mungojam** [comment]
this affects https://www.nuget.org/packages/Amazon.Extensions.Configuration.SystemsManager/

It feels like more than a feature request

**Comment by thesuavehog** [comment]
Yes, there are multiple extensions that need updating.
I didn't pore through the code to see if the version bump breaks anything, but when the AWSSDK.Core library is updated, the first-party packages that depend on it need to get updated as fast as possible to support it (in theory, the Product/Project Manager for AWSSDK.Core should already know what first-party libraries will work or not with the updated version...).

**Comment by zachsawcs** [comment]
That's more common sense than a feature request isn't it?

**Comment by doug-ferris** [comment]
I opened a PR (#75) , #73  would probably cover this better, but I didn't want to bleed the addition of the STS package with simple package updates, so I opened a new PR (which I am happy to have rejected if that is not the way we wanted this to go)

**Comment by tenacioustechie** [comment]
Any update or timline on this? or #75 ? This is blocking me updating packages as well.

**Comment by ashovlin** [comment]
Just released #75 in version 2.0.0, thanks.
